src/main/java/teammates/ui/template/StudentProfile.java

```diff
@@ -36,7 +42,7 @@ public String getShortName() {
     }
 
     public String getGender() {
-        return gender;
+        return gender.name();
```

Comment:
Getters generally should be a simple `return`. Why not make the return type `Gender`? There is no prescription that the return type needs to be `String`.